invideoquiz/public/js/src/invideoquiz.js

```diff
@@ -44,6 +45,15 @@ function InVideoQuizXBlock(runtime, element) {
 
     function setUpStudentView(component) {
         var componentIsVideo = component.data('id').indexOf(videoId) !== -1;
+        if (videoType === 'video_xblock') {
```

Comment:
Please add comments to newly created functionality, so we will be aware of code part with video_xblock support.